modules/ROOT/pages/about-flows.adoc

```diff
@@ -42,20 +37,27 @@ out to other services, you might factor out that behavior into other flows.
 
 == Using Subflows
 
-You can also group a series of event processors into subflows. Like a flow, a
-subflow is a scope that groups together a sequence of event processors, but it
-does not have an event source, and it does not have its own error handling.
-Subflows are always called from another flow or subflow through a Flow Ref
-component.
+A subflow is a scope that enables you to group event processors, similar to a flow, but with certain differences and limitations:
+
+* Subflows do not have event sources.
+* Subflows do not have an error handling scope.
+* During design, subflows work as a macro that replaces the Flow Reference components that call the subflow. When you build the application, all Flow Reference components that call a subflow are replaced by the contents of the referenced subflow.
+* Referencing a subflow has higher performance than referencing a flow.
+
+=== Considerations
+
+Because the contents of the subflow replace each Flow Reference component that references the subflow, several instances of the event processors inside the subflow exist in the application at runtime. This behavior can cause issues with event processors that depend on unique IDs or instances to execute like Batch processors.
```

Comment:
Is it obvious to your audience WHY replacing references to a subflow with its contents results in many event processors existing in the app at runtime, and why that affects processors that depend on unique IDs? Do the several processors that result NOT have unique IDs?
Oh. Nevermind. I see it is explained below.

Reply:
```suggestion
Because the contents of a subflow replace each Flow Reference component that references that subflow, several instances of the event processors inside that subflow exist in the application at runtime. This behavior can cause issues with event processors that depend on unique IDs or instances to execute, such as batch processors.
```